.husky/pre-commit

```diff
@@ -1,2 +1,2 @@
-npx pretty-quick --staged
-npx --no-install lint-staged
+pnpx pretty-quick --staged
+pnpx lint-staged
```

Comment:
Was `pnpx` niet on the way out? Zouden we `pnpm exec` moeten gebruiken?

Reply:
Het is me niet meer helemaal duidelijk.

`pnpm foo` draait ook gewoon `node_modules/.bin/foo` als die bestaat en er is ook nog `pnpm dlx` (https://pnpm.io/cli/dlx)

Reply:
Ik kan inderdaad niet echt meer iets over pnpx vinden; ik denk dat `pnpm exec` de weg vooruit is voor packages die (dev)dependency zijn.